content/data-reporting/changelog/index.mdx

```diff
@@ -11,7 +11,21 @@ This document is the record of all the changes in [**the Reports API**](/data-re
 
 ## [v3.4] - Developer preview
 
-No changes yet.
+- POST method becomes primary and recommended way of sending requests. GET method is still supported, however due to limitations of query string length might fail for more complex requests.
+- Request format has a new format, the filters are nested within the `filters` field now.
+- To provide more consistent experience across Platform APIs, filters were reworked and aligned with those used in the **List Archives** ([Web](/messaging/agent-chat-api/v3.4/#list-archives). All reports:
+  - had `from` field renamed to `filters.from`
+  - had `to` field renamed to `filters.to`
+  - had `groups` field renamed to `filters.groups.values`
+  - had `agents` field renamed to `filters.agents.values`
+  - had `agent_assigned` field renamed to `filters.agents.exists`
+  - had `names` field renamed to `filters.tags.values`
+  - had `names:with` field renamed to `filters.tags.exists`
+  - had `names:without` field renamed to `filters.tags.exists`
```

Comment:
Is this correct? If so, maybe join these 2 points? (And also the old `tagged` filter?)

Reply:
It is correct, with translates into exists: true and without into exists: false

Reply:
So if I understand it correctly, `names:with` and `tagged` are equivalent, right? I would mark that in the changelog

Reply:
Yes `names:with` is equivalent of `tagged`, both are used in different reports in 3.2 & 3.3. But these are old filters, not compatible with 3.4 anymore, i believe that mapping old => new is enough